pallets/orml-currencies-allowance-extension/src/benchmarking.rs

```diff
@@ -7,7 +7,10 @@ use sp_std::prelude::*;
 benchmarks! {
 	add_allowed_currencies {
```

Comment:
Not so sure if this helps: I changed the implementation of `add_allowed_currencies` for this one:

```rust
add_allowed_currencies {
		let native_currency_id = <T as orml_currencies::Config>::GetNativeCurrencyId::get();
		let x in 1..T::MaxAllowedCurrencies::get();
	}: add_allowed_currencies(RawOrigin::Root, vec![native_currency_id; x as usize])
	verify {
		let native_currency_id = <T as orml_currencies::Config>::GetNativeCurrencyId::get();
		assert_eq!(AllowedCurrencies::<T>::get(native_currency_id), Some(()));
	}
```
After this change, when running the command indicated in the PR description, I obtained this default_weights.rs file and this output (both files are compressed in the results zip file).

[results.zip](https://github.com/pendulum-chain/pendulum/files/12135702/results.zip)

Reply:
Amazing, this helped a lot, thanks 🙏